### ISSUE TITLE
Need help with setup

### ISSUE THREAD

**Issue by lownotelee** [artnavsegda/homebridge-coolmaster#1]
**Describe Your Problem:**

There are no instructions on how to set up config.json. I tried looking through the code to see what's expected and have filled it out, but I have a feeling the port hard coded in the plugin is wrong; I think it's meant to be 10102.

Using Postman, I tried 10102 with the rest of the request url in the code, and it's returned a malformed response

**Logs:**

```
This plugin threw an error from the characteristic 'Current Heater-Cooler State': Unhandled error thrown inside read handler for characteristic: request to http://192.168.1.171:10103/v2.0/device/283B960046A4/ls2&L1.005 failed, reason: connect ECONNREFUSED 192.168.1.171:10103. See https://git.io/JtMGR for more info.
```

**Plugin Config:**
serial removed
```json
{
            "name": "CoolMasterNet Dynamic Platform",
            "platform": "CoolMasterHomebridgePlugin",
            "ip": "192.168.1.171",
            "serial": "xxxxxxxxxxx",
            "accessories": [
                {
                    "uniqueId": "L1.005",
                    "displayName": "ac"
                }
            ]
        }
```

**Screenshots:**

**Environment:**

* **Plugin Version**: latest
* **Homebridge Version**: 1.3.4
* **Node.js Version**:  14.15.4
* **NPM Version**:  6.14.11
* **Operating System**:  Raspbian

**Comment by netanelassor** [comment]
have you make it work?

**Comment by lownotelee** [comment]
No, but I did find another plugin by Cool Automation that was written by the manufacturer themselves 

https://www.npmjs.com/package/homebridge-cool-automation

**Comment by artnavsegda** [comment]
Sorry for being late for the party, but anyways

To perform correctly,  coolmaster itself has to be configured to receive REST control, by the 10103 port, details by the https://coolautomation.com/wp-content/uploads/sites/2/2019/09/CoolMasterNet-REST-API-spec.pdf manual

And yes, this plugin has long been dead in development, however, primary intentions was to use REST port since default port blocks control by the different entities, including [homebridge-cool-automation](https://www.npmjs.com/package/homebridge-cool-automation)

REST api, however are non-blocking
